packages/vuelidate/src/core.js

```diff
@@ -323,9 +316,9 @@ function createMetaFields (results, ...otherResults) {
 
     // collect all nested and child $errors
     const nestedErrors = allResults.value
-      .filter(result => result.$errors.length)
+      .filter(result => result.$errors.value.length)
       .reduce((errors, result) => {
-        return errors.concat(...result.$errors)
+        return errors.concat(...result.$errors.value)
```

Comment:
If `$errors` is an array, I don’t think we need to spread it to use it with concat.

Reply:
I think you are right